cdap-app-templates/cdap-etl/cdap-etl-tools/src/main/java/co/cask/cdap/etl/tool/UpgradeTool.java

```diff
@@ -104,20 +120,144 @@ private boolean upgrade(Id.Application appId) throws Exception {
       return false;
     }
 
+    System.out.println("Upgrading pipeline: " + appId);
```

Comment:
Would it be better to use a logger here that appends to stdout instead? Although that can be done separately too.